helpers/exceptions.js

```diff
@@ -15,15 +15,26 @@
 'use strict';
 
 /**
+ * Description of the namespace.
+ *
  * @namespace exceptions
- * @memberof module:helpers
+ * @memberof helpers
+ * @see Parent: {@link helpers}
+ * @property {Array} blockRewards
+ * @property {Array} delegates
  * @property {Object} genesisPublicKey
  * @property {string} genesisPublicKey.mainnet
  * @property {string} genesisPublicKey.testnet
+ * @property {Object} rounds
+ * @property {Object} rounds.27040
+ * @property {number} rounds.27040.rewards_factor
+ * @property {number} rounds.27040.fees_factor
+ * @property {number} rounds.27040.fees_bonus
  * @property {string[]} senderPublicKey
```

Comment:
As previously with arrays.

Reply:
this will be part of a new issue for standardizing typedefs, as discussed